aggregator/fetchers/xexchange.go

```diff
@@ -35,32 +35,32 @@ type graphqlResponse struct {
 	} `json:"data"`
 }
 
-type maiar struct {
```

Comment:
maybe add a TODO on L15 to not forget to update the query after the rebranding work is done on the data-api?

Reply:
pushed